drw_base.h

```diff
@@ -101,6 +101,22 @@ enum DBG_LEVEL {
 
 //! Special codes for colors
 enum ColorCodes {
+    Black = 250,
```

Comment:
Can you sort the colors by their numeric value? That'd make it easier to add more in the future.